Index: ruby-rmagick/post-setup.rb
===================================================================
--- ruby-rmagick.orig/post-setup.rb	2011-06-07 01:29:12.670529194 +0200
+++ ruby-rmagick/post-setup.rb	2011-06-07 01:29:18.750529196 +0200
@@ -236,7 +236,7 @@
     examples -= DONT_RUN
     es = ExampleSet.new(examples.length)
     begin
-      examples.each { |example| es.update(example) }
+      examples.each { |example| es.build(example) }
     ensure
       es.persist
     end
