Index: ruby-rmagick/ext/RMagick/rmmain.c
===================================================================
--- ruby-rmagick.orig/ext/RMagick/rmmain.c	2011-05-17 12:20:11.000000000 -0500
+++ ruby-rmagick/ext/RMagick/rmmain.c	2011-05-17 12:54:43.000000000 -0500
@@ -1639,6 +1639,11 @@
     int x, n;
     ID bypass = rb_intern("RMAGICK_BYPASS_VERSION_TEST");
 
+    /* Vincent Fourmond: disabled by default on Debian as it is
+       obnoxious and unnecessary when appropriate SONAME is set for
+       ImageMagick libraries (which we should assume anyway) */
+    return;
+
     if (RTEST(rb_const_defined(rb_cObject, bypass)) && RTEST(rb_const_get(rb_cObject, bypass)))
     {
         return;
