Description: fix missing MagickLibSubversion symbol
Origin: upstream, slightly trimmed version of commit: http://github.com/rmagick/rmagick/commit/70dfc29c883588a6bf7d609dd70193f3ce49509d
Index: librmagick-ruby-2.13.0/ext/RMagick/extconf.rb
===================================================================
--- librmagick-ruby-2.13.0.orig/ext/RMagick/extconf.rb	2009-12-24 07:03:02.000000000 +0100
+++ librmagick-ruby-2.13.0/ext/RMagick/extconf.rb	2010-03-15 20:27:17.822392038 +0100
@@ -233,6 +233,7 @@
    "LevelColorsImageChannel",        # 6.5.6-4
    "LevelizeImageChannel",           # 6.4.2
    "LiquidRescaleImage",             # 6.3.8-2
+   "MagickLibAddendum",              # 6.5.9-1
    "OpaquePaintImageChannel",        # 6.3.7-10
    "QueueAuthenticPixels",           # 6.4.5-6
    "RemapImage",                     # 6.4.4-0
Index: librmagick-ruby-2.13.0/ext/RMagick/rmagick.h
===================================================================
--- librmagick-ruby-2.13.0.orig/ext/RMagick/rmagick.h	2009-12-21 11:34:56.000000000 +0100
+++ librmagick-ruby-2.13.0/ext/RMagick/rmagick.h	2010-03-15 20:27:17.822392038 +0100
@@ -162,6 +162,13 @@
 #define DestroyConstitute(void) ConstituteComponentTerminus(void)
 #endif
 
+/** ImageMagick 6.5.9 replaced MagickLibSubversion with
+ * MagickLibAddendum.
+ */
+#if defined(HAVE_MAGICKLIBADDENDUM)
+#define MagickLibSubversion MagickLibAddendum
+#endif
+
 /** IM 6.4.1 replaced AllocateImage with AcquireImage.
  * Both have the same signature.
  */
